webapp/src/store/ProjectStore.spec.ts

```diff
@@ -164,31 +164,6 @@ describe('ProjectStore', () => {
     await expect(actual).rejects.toThrowError(LanguageNotFound);
   });
 
-  it('throws exception for unknown message ID', async () => {
```

Comment:
I'm fine with removing this. I know we discussed it at the hackathon and at the time I didn't want to change both the application logic and the tests, but I agree it's not necessary for this to throw an exception. 😊